data/ui/oops-window.ui

```diff
@@ -205,455 +188,251 @@
+                            <property name="spacing">12</property>
+                            <child>
+                              <object class="GtkLabel" id="lbl_reported_value">
+                                <property name="no_show_all">True</property>
+                                <property name="halign">start</property>
+                                <property name="label">no</property>
+                                <property name="selectable">True</property>
+                              </object>
+                            </child>
+                          </object>
+                          <packing>
+                            <property name="left_attach">1</property>
+                            <property name="top_attach">3</property>
                           </packing>
                         </child>
                       </object>
+                    </child>
+                    <child>
+                      <object class="GtkBox" id="vbx_problem_messages">
+                        <property name="visible">True</property>
+                        <property name="vexpand">True</property>
+                        <property name="orientation">vertical</property>
+                      </object>
+                    </child>
+                    <child>
+                      <object class="GtkButton" id="btn_detail">
+                        <property name="label" translatable="yes">D_etails</property>
+                        <property name="tooltip_text" translatable="yes">Show problem details</property>
+                        <property name="use-underline">True</property>
+                        <property name="action-name">win.details</property>
+                        <property name="visible">True</property>
+                        <property name="can_focus">True</property>
+                        <property name="receives_default">True</property>
+                        <property name="halign">end</property>
+                      </object>
+                    </child>
+                  </object>
+                </child>
+                <child>
+                  <object class="GtkBox" id="vbx_empty_page">
+                    <property name="visible">True</property>
+                    <property name="orientation">vertical</property>
+                    <child>
+                      <object class="GtkLabel" id="lbl_no_oopses">
+                        <property name="visible">True</property>
+                        <property name="hexpand">True</property>
+                        <property name="vexpand">True</property>
+                        <property name="label" translatable="yes">No problems detected!</property>
+                        <property name="selectable">True</property>
+                        <style>
+                          <class name="dim-label"/>
+                        </style>
+                      </object>
```

Comment:
It's not related with this commit but I think this label looks sad and needs a minor redesign. Maybe it should be larger. @allanday, what do you think?

Explanation: This is the label displayed when there are no bugs detected, e.g., in a newly installed system.

Reply:
The HIG has some guidelines for initial/empty states (https://developer.gnome.org/hig/stable/initial-state-placeholder.html.en and https://developer.gnome.org/hig/stable/empty-placeholder.html.en, for instance), but I can’t see anything about specific text sizes.

I think we could just pick a few apps (e.g. Nautilus, with its empty directory state), and draw inspiration.